### PR TITLE
Realese

### PR DESCRIPTION
**Pull request by PeterKuzmenko**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,12 +12,15 @@
           <div class="stackable grid">
             <div class="row">
               <div class="three wide center aligned column">
-                <!-- Avatar -->
+                <img src="image/754786788159695.jpg" width="150px" height="150px">
```
Comment (nick-mychka) [inline index.html]: img tag need alt attribute

Comment (PeterKuzmenko) [reply index.html]: Ok, I will make this now

File: index.html [hunk 2]
```diff
@@ -12,12 +12,15 @@
...
               </div>
               <div class="thirteen wide column">
                 <div class="stackable grid">
                   <div class="sixteen wide column">
-                    <div class="job">President of United States of America</div>
+                    <div class="job">
+                        <h1>Donald Trump</h1>
+                      <p>President of United States of American</p>
```
Comment (nick-mychka) [inline index.html]: Need fix typo 'American' to 'America'

Comment (PeterKuzmenko) [reply index.html]: Ok, I will make that too

**Pull request merged**